--- SDL2-2.0.9.orig/Makefile	2018-01-29 07:12:13.000000000 +0100
+++ SDL2-2.0.9/Makefile	2019-01-21 18:20:28.390884000 +0100
@@ -5,15 +5,15 @@
 ARCHITECTURES := i686-w64-mingw32 x86_64-w64-mingw32
 
 all install:
-	@echo "Type \"make native\" to install 32-bit to /usr"
-	@echo "Type \"make cross\" to install 32-bit and 64-bit to $(CROSS_PATH)"
+	@echo "Type \"$(MAKE) native\" to install 32-bit to /usr"
+	@echo "Type \"$(MAKE) cross\" to install 32-bit and 64-bit to $(CROSS_PATH)"
 
 native:
-	make install-package arch=i686-w64-mingw32 prefix=/usr
+	$(MAKE) install-package arch=i686-w64-mingw32 prefix=/usr
 
 cross:
 	for arch in $(ARCHITECTURES); do \
-	    make install-package arch=$$arch prefix=$(CROSS_PATH)/$$arch; \
+	    $(MAKE) install-package arch=$$arch prefix=$(CROSS_PATH)/$$arch; \
 	done
 
 install-package:
